packages/styleguide/src/components/ExpandableLink/ExpandableLinkCallout.tsx

```diff
@@ -0,0 +1,241 @@
+import React, { useMemo, useEffect, useRef, useState } from 'react'
+import { css, keyframes } from 'glamor'
+import { mUp } from '../../theme/mediaQueries'
+import zIndex from '../../theme/zIndex'
+import { useColorContext } from '../Colors/useColorContext'
+import RawHtml from '../RawHtml'
+import { LinkIcon } from '../Icons'
+import { fontStyles } from '../Typography'
+import { ellipsize } from '../../lib/styleMixins'
+import { StateProps, DELAY } from './ExpandableLink'
+
+type Props = {
+  inNativeApp?: boolean
+  timeOutRef: React.MutableRefObject<NodeJS.Timeout>
+  expandedLink: StateProps
+  setExpandedLink: React.Dispatch<React.SetStateAction<StateProps>>
+}
+
+const ExpandableLinkP = ({ children, ...props }) => {
+  const [colorScheme] = useColorContext()
+  return (
+    <p {...props} {...styles.contentText} {...colorScheme.set('color', 'text')}>
+      {children}
+    </p>
+  )
+}
+
+const fadeIn = keyframes({
+  from: {
+    opacity: 0,
+  },
+  to: {
+    opacity: 1,
+  },
+})
+
+const shortenLink = (href) => {
```

Comment:
While I do not attempt to understand this code the suffix of `/.../` does not seem useful to me in the following examples:

<img width="542" alt="Screenshot 2022-07-25 at 18 19 39" src="https://user-images.githubusercontent.com/410211/180826702-75026333-ed93-413c-9693-2eaf732cbb80.png">
<img width="545" alt="Screenshot 2022-07-25 at 18 19 35" src="https://user-images.githubusercontent.com/410211/180826709-c2b6b5d9-f121-42eb-8eae-290a020fc991.png">

Full urls:
https://eatforum.org/eat-lancet-commission/
https://www.quarks.de/umwelt/klimawandel/darum-sollten-wir-ueber-methan-sprechen/